Rules/CommonScripts/Holiday/Christmas/Christmas.as

```diff
@@ -54,8 +56,7 @@ void onTick(CRules@ this)
 		}
 	}
 	
-	
```

Comment:
removed line causes diff

Reply:
ok, I guess it was redundant though

Reply:
nothing to resolve but im resolving the conversation